src/virtio-pci.h

```diff
@@ -7,6 +7,7 @@
 
 #define VIRTIO_PCI_VENDOR_ID 0x1AF4
 #define VIRTIO_PCI_DEVICE_ID_BLK 0x1042
+#define VIRTIO_PCI_DEVICE_ID_NET 0x1041
```

Comment:
Swap the order between `VIRTIO_PCI_DEVICE_ID_NET` and `VIRTIO_PCI_DEVICE_ID_BLK`.

Reply:
Yes, I have already swapped the order.